### PR TITLE
feat(core): Add Clear all filters to grid menu

### PR DESCRIPTION
**Pull request by mboriani**
I add a new menu label action call "Clear all filters". This action clear all existing filters in the grid. To enable you have to have enableFiltering = true.

### CODE REVIEW

**Comment by swalters** [comment]
Please don't commit files with only whitespace changes

**Comment by JLLeitschuh** [comment]
Sometimes you don't really have a choice.
Please fix your code to make sure that the tests don't fail.

**Comment by mboriani** [comment]
Look like when i run grunt karmangular --browsers=Chrome it is failing that test but on the opposite way.

**Comment by JLLeitschuh** [comment]
Ah. I see what @swalters was referring to. Your commit has whitespace changes in files you didn't otherwise modify. Can you please remove those files from your PR and squash them into one?
Thanks!

File: src/features/cellnav/test/uiGridCellNavService.spec.js [hunk 1]
```diff
@@ -290,8 +290,8 @@ describe('ui.grid.edit uiGridCellNavService', function () {
...
       });
       $timeout.flush();
-      
+
       expect(args).toEqual( null );
     });
   });
-});
\ No newline at end of file
+});
```
Comment (JLLeitschuh) [inline src/features/cellnav/test/uiGridCellNavService.spec.js]: The only changes to this file are whitespace changes. Please remove from your PR.

**Pull request merged**